rqt_gps_rtk_plugin/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>rqt_gps_rtk_plugin</name>
+  <version>0.0.1</version>
+  <description>The rqt_gps_rtk_plugin package</description>
+
+  <maintainer email="kaiho@ethz.ch">Kai Holtmann</maintainer>
+
+  <license>Proprietary</license>
```

Comment:
No "BSD"?

Reply:
Obviously BSD (since it's open source). Just forgot to update the license tag ;-)